newIDE/app/package.json

```diff
@@ -100,6 +101,8 @@
     "extract-all-translations": "node scripts/extract-all-translations.js",
     "compile-translations": "node scripts/compile-translations.js",
     "reload-extensions": "node scripts/import-GDJS-Runtime.js",
+    "build-theme-resources": "node scripts/compile-theme-styles.js",
```

Comment:
Let's use the same name for both the script and the command name: "build-theme-resources"
(compile sounds a bit like preprocessing files à la Sass, LESS, while here it's about generating json files).

Reply:
done ✅